ingestion-beam/src/main/java/com/mozilla/telemetry/options/InputType.java

```diff
@@ -24,6 +25,26 @@ public Read read(SinkOptions.Parsed options) {
     public Read read(SinkOptions.Parsed options) {
       return new FileInput(options.getInput(), options.getInputFileFormat());
     }
+  },
+
+  bigquery_table {
+
+    /** Return a PTransform that reads from a BigQuery table. */
+    public Read read(SinkOptions.Parsed options) {
+      return new BigQueryInput(options.getInput(), options.getBqReadMethod(),
+          BigQueryInput.Source.TABLE, options.getBqRowRestriction(), options.getBqSelectedFields());
+    }
+
+  },
+
+  bigquery_query {
+
+    /** Return a PTransform that reads the results of a BigQuery query. */
+    public Read read(SinkOptions.Parsed options) {
+      return new BigQueryInput(options.getInput(), options.getBqReadMethod(),
+          BigQueryInput.Source.QUERY, options.getBqRowRestriction(), options.getBqSelectedFields());
+    }
```

Comment:
I chose to implement 2 new input types rather than a single `bigquery` input type, allowing us to distinguish between reading from a table vs. a query without introducing an additional compile-time parameter.